article-store/Pipfile

```diff
@@ -10,12 +10,14 @@ name = "pypi"
 django = "~=2.0.5"
 django-debug-toolbar = "~=1.9.1"
 djangorestframework = "~=3.8.2"
-psycopg2-binary = "*"
+"psycopg2-binary" = "*"
```

Comment:
pipenv strikes back - not a problem

Reply:
https://github.com/pypa/pipenv/issues/1933